### PR TITLE
Class branch

### PR DESCRIPTION
**Pull request by Roseokpe**
-  we refactored the Awesome books application to use JavaScript classes.
- Added styling to make it look like the wireframe
- displayed the value of what the user puts in and store it to local storage
- Added a remove button to remove the specific element the user clicks

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @NickEmma,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## **Invalid Code Review Request**

You have submitted a project, that _**you have submitted a PR without description about the project specs, please add the description then re-submit for review**_

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+// ============ UI class: Handle ui task ============
+class UI {
```
Comment (Ismailco) [inline app.js:40]: - [ ] The website should be as the wireframe, kindly modify the UI so the general design looks like the wireframe

|Your design|The wireframe|
|-|-|
|Screenshot_20220628_191613|awesome_books_core_elements (1)|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @NickEmma, @Roseokpe,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NickEmma** [comment]
No sir this wasn't the state of design it's not plain like this

File: app.js (hunk 1 above)
Comment (NickEmma) [reply app.js:40]: This is not the state of my design

Comment (Ismailco) [reply app.js:40]: @NickEmma, This is the site that I get when running live server

Comment (NickEmma) [reply app.js:40]: let me check it again at my end

Comment (NickEmma) [reply app.js:40]: you are running it on the main branch @Ismailco you should try the class branch

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Roseokpe , 

Your project is complete!  Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**